scripts/fullTest.py

```diff
@@ -126,7 +126,7 @@ def runTests(testNames, categoryName, marshalArgs=[], cmdArgs=[]):
 
     for tName in testNames:
         log.log(logging.INFO, "[{}] {}:".format(categoryName, tName))
-        tPath = testDir / (tName + ".json")
+        tPath = testDir / (tName + ".yaml")
```

Comment:
It seems that all `test/*.json` files should be renamed to `*.yaml`, but they don't appear to be committed yet.  I see a bunch of file deletions in the diff but no moves.

Reply:
derp. fixed